README.md

```diff
@@ -22,6 +22,6 @@ Please note that this *does not cover* the usage of the "osu!" or "ppy" branding
 
 Contributions to the osu!wiki are now done through GitHub.
 
-A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [here](/wiki/osu!wiki_contribution_guide).
+A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [](/wiki/osu!wiki_contribution_guide).
 
 The article style guide, which _must_ be adhered to for all new and rewritten pages going forward, is available [here](/wiki/Article_Style_Guide).
```

Comment:
Why not fix this too while you were at it...? replace with `[Article Style Guide](/wiki/Article_Style_Guide).`